### DISCUSSION TITLE
Add bookmarks and maybe notes

### DISCUSSION THREAD

**Discussion by uniquename** [gotson/komga#1566] [Ideas]
### Describe your suggested feature

I thought it might be useful to add a bookmark and maybe a notes feature.

It is great that the current page is saved, but for example for anthologies it would be nice to be able to create bookmarks and also to mark chapters. 

Maybe that should be another issue, but in addition it could help to also be able to add some text to those bookmarks or as notes.

I've searched the issue queue, but couldn't find anything about that, hope I'm not creating a duplicate.

cheers and great software btw! 

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to the latest version.
- [X] I will fill out all of the requested information in this form.

**Comment by da-the-dev** [comment]
+1
